x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/search/SearchApplicationTemplate.java

```diff
@@ -30,6 +31,8 @@
  * {@link QuerySearchApplicationAction}, overriding the parameters included on it via {@link Request}
  */
 public class SearchApplicationTemplate implements ToXContentObject, Writeable {
+
+    public static final SearchApplicationTemplate DEFAULT_TEMPLATE = buildDefaultTemplate();
```

Comment:
nit: we probably don't need a function and can set the definition inline with the final variable?